docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug: `1247` Fix glitchy autocomplete components which were not opening properly if the "dropdown arrows" were cliked.
```

Comment:
cliked -> clicked

It's not autocomplete components though, right? Isn't it filtering components?

Reply:
The bug was in properties supplied to `<v-autocomplete>`, which is a building block for various "select" menus we have in the app (tag filters, the general asset select, etc.).

Reply:
All right I see. So it affected more than just the reported locations. nice that it's fixed then!

Reply:
@isidorosp do you want to fix the typo here? You can [skip ci]

Reply:
Fixed in newest commit & made the change log entry more detailed.

Reply:
Great @isidorosp. Merging.